<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1091</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>特装型课表MKV</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="RightPanelFrame">
    <property name="geometry">
     <rect>
      <x>1690</x>
      <y>110</y>
      <width>201</width>
      <height>861</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <widget class="QLabel" name="classIndicator">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>821</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>汉仪文黑-85W</family>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgba(255, 255, 255, 242);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;→&lt;/p&gt;&lt;p&gt;→&lt;/p&gt;&lt;p&gt;→&lt;/p&gt;&lt;p&gt;→&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="classTable">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>111</width>
       <height>821</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>汉仪文黑-85W</family>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgba(255, 255, 255, 242);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;语文&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;数学&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;英语&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;物理&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;化学&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;生物&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;政治&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;历史&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;地理&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="classBg">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>201</width>
       <height>861</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px rgba(31, 31, 31, 191);border-radius:10px;padding:0px 4px;background-color:rgba(31, 31, 31, 191);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>classBg</zorder>
    <zorder>classIndicator</zorder>
    <zorder>classTable</zorder>
   </widget>
   <widget class="QFrame" name="LeftPanelFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>1081</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="Duty_Frame">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>200</y>
       <width>361</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgba(255, 255, 255,250)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <widget class="QLabel" name="dutyLabel1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>字魂59号-创粗黑</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>组长</string>
      </property>
     </widget>
     <widget class="QLabel" name="dutyLeader">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>-5</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>李旭科书法 v1.4</family>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>陶学明</string>
      </property>
     </widget>
     <widget class="QLabel" name="dutyLabel2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>字魂59号-创粗黑</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>扫地</string>
      </property>
     </widget>
     <widget class="QLabel" name="dutySweepA">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>-5</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>李旭科书法 v1.4</family>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>苗梦娇</string>
      </property>
     </widget>
     <widget class="QLabel" name="dutyLabel3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>字魂59号-创粗黑</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>拖地</string>
      </property>
     </widget>
     <widget class="QLabel" name="dutyMop">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>45</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>李旭科书法 v1.4</family>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>徐丹</string>
      </property>
     </widget>
     <widget class="QLabel" name="dutyLabel4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>字魂59号-创粗黑</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>黑板</string>
      </property>
     </widget>
     <widget class="QLabel" name="dutyBoard">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>95</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>李旭科书法 v1.4</family>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>沈红杰</string>
      </property>
     </widget>
     <widget class="QLabel" name="dutySweepB">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>45</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>李旭科书法 v1.4</family>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>周学红</string>
      </property>
     </widget>
     <widget class="QLabel" name="dutySweepC">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>95</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>李旭科书法 v1.4</family>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>卢建武</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="DateTimeFrame">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>361</width>
       <height>91</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="timeNow">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>211</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DIN1451</family>
        <pointsize>58</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgba(255, 255, 255,250);letter-spacing:4px</string>
      </property>
      <property name="text">
       <string>20:20</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="wkdayNow">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>49</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>字魂59号-创粗黑</family>
        <pointsize>23</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgba(255, 255, 255,250);letter-spacing:-1px</string>
      </property>
      <property name="text">
       <string>星期日</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="dateNow">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>18</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rajdhani Medium</family>
        <pointsize>23</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgba(255, 255, 255,250);letter-spacing:0px</string>
      </property>
      <property name="text">
       <string>08 / 10</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Timer" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>460</y>
       <width>391</width>
       <height>151</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="_remains_3">
      <property name="geometry">
       <rect>
        <x>111</x>
        <y>59</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>字魂59号-创粗黑</family>
        <pointsize>28</pointsize>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);letter-spacing:-3px;</string>
      </property>
      <property name="text">
       <string>还有</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="timeDescription">
      <property name="geometry">
       <rect>
        <x>109</x>
        <y>99</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DIN1451</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255,255,255);text-transform:uppercase;letter-spacing:0.5px;</string>
      </property>
      <property name="text">
       <string>the next point approaches</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="indent">
       <number>7</number>
      </property>
     </widget>
     <widget class="QLabel" name="timeConsumption">
      <property name="geometry">
       <rect>
        <x>109</x>
        <y>119</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DIN1451</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255,255,255);text-transform:uppercase;letter-spacing:0.5px;</string>
      </property>
      <property name="text">
       <string>IN 114 minutes</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="indent">
       <number>7</number>
      </property>
     </widget>
     <widget class="QLabel" name="timeTitle">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>19</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>字魂59号-创粗黑</family>
        <pointsize>28</pointsize>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);letter-spacing:-3px</string>
      </property>
      <property name="text">
       <string>下个时间点</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="CountDown" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>630</y>
       <width>391</width>
       <height>151</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="_remains">
      <property name="geometry">
       <rect>
        <x>111</x>
        <y>59</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>字魂59号-创粗黑</family>
        <pointsize>28</pointsize>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);letter-spacing:-3px;</string>
      </property>
      <property name="text">
       <string>还有</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="cdDescription">
      <property name="geometry">
       <rect>
        <x>109</x>
        <y>99</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DIN1451</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255,255,255);text-transform:uppercase;letter-spacing:0.5px;</string>
      </property>
      <property name="text">
       <string>THE NEMT STARts</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="indent">
       <number>7</number>
      </property>
     </widget>
     <widget class="QLabel" name="cdConsumption">
      <property name="geometry">
       <rect>
        <x>109</x>
        <y>119</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DIN1451</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255,255,255);text-transform:uppercase;letter-spacing:0.5px;</string>
      </property>
      <property name="text">
       <string>IN 114 DAYS</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="indent">
       <number>7</number>
      </property>
     </widget>
     <widget class="QLabel" name="cdTitle">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>19</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>字魂59号-创粗黑</family>
        <pointsize>28</pointsize>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);letter-spacing:-3px</string>
      </property>
      <property name="text">
       <string>距高考开始</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="Formula_Frame">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>780</y>
       <width>541</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgba(255, 255, 255,250)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <widget class="QTextBrowser" name="formula">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>541</width>
        <height>271</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;border:0px</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="acceptRichText">
       <bool>true</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::NoTextInteraction</set>
      </property>
      <property name="openLinks">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="RedFrame">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>460</y>
      <width>391</width>
      <height>321</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="timeRemain">
     <property name="geometry">
      <rect>
       <x>183</x>
       <y>12</y>
       <width>201</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DIN1451</family>
       <pointsize>64</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgba(255, 21, 0, 240);</string>
     </property>
     <property name="text">
      <string>88</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="cdDigit">
     <property name="geometry">
      <rect>
       <x>183</x>
       <y>180</y>
       <width>131</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DIN1451</family>
       <pointsize>64</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgba(255, 21, 0, 240);</string>
     </property>
     <property name="text">
      <string>123</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="timeLine">
     <property name="geometry">
      <rect>
       <x>104</x>
       <y>61</y>
       <width>5</width>
       <height>75</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 21, 0, 240);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="cdLine">
     <property name="geometry">
      <rect>
       <x>104</x>
       <y>230</y>
       <width>5</width>
       <height>75</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 21, 0, 240);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <zorder>LeftPanelFrame</zorder>
   <zorder>RightPanelFrame</zorder>
   <zorder>RedFrame</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
